src/guide/backporting.md

```diff
@@ -7,19 +7,19 @@
 
 Any change that originally required a CSR will require a new CSR to be backported unless the backport was covered by the initial CSR. Changes to Java SE specifications cannot be made in an update release without a Java SE Maintenance Release. CSR-related issues affect interfaces and behavior and must be very carefully scrutinized.
 
-## Backporting to a feature release stabilization repository
+## Backporting to a feature release stabilization branch
 
-During rampdown of a feature release there are two repositories in play, the release stabilization repository for the outgoing release, and the mainline repository where the next release is being developed. Any change going into the release stabilization repository is likely to be desired in mainline as well. When making a change intended both for the stabilization release and the mainline repository, you should always create your pull request targeting the mainline repository first, and then, once the pull request is integrated, backport the resulting commit to the release stabilization repository. For bugfixes that are **only** applicable to the release stabilization repository, regular pull requests targeting the stabilization fork should be created.
+During ramp down of a feature release there are two branches of the mainline repository in play, the release stabilization branch for the outgoing release, and the master branch where the next release is being developed. Any change going into the release stabilization branch is likely to be desired in the master branch as well. When making a change intended for both the stabilization and the master branches, you should always create your pull request targeting the master branch first, and then, once the pull request is integrated, backport the resulting commit to the release stabilization branch. For bug fixes that are **only** applicable to the release stabilization branch, regular pull requests targeting the stabilization branch should be created.
 
-Please note that special rules applies during rampdown regarding what can and can't be included into the rampdown repository. See the [The JDK Release Process] for more information.
+Please note that special rules applies during ramp down regarding what can and can't be included into the ramp down branch. See the [The JDK Release Process] for more information.
```

Comment:
Instead of "ramp down branch", I might use "stabilization branch" as you do in the previous paragraph.

Reply:
Agreed. Fixed.